build-system/tasks/helpers.js

```diff
@@ -100,7 +100,7 @@ const BABELIFY_GLOBAL_TRANSFORM = {
  * Plugins used by Babelify while compiling unminified code
  */
 const BABELIFY_PLUGINS = {
-  plugins: [conf.getReplacePlugin(), conf.getJsonConfigurationPlugin()],
+  plugins: [conf.getReplacePlugin(), conf.getJsonConfigurationPlugin(), conf.getRewritePlugin()],
```

Comment:
Sanity check: The plugins in `BABELIFY_PLUGINS` will be applied to all files during `gulp build`, `gulp unit`, and `gulp integration`. I assume `rewrite-imports` is a no-op for files that aren't in `node_modules`?

Reply:
No, it will rewrite any import that matches.

Reply:
Never mind, I don't see why this is a problem.